src/test/java/org/cactoos/scalar/IoCheckedTest.java

```diff
@@ -29,18 +29,18 @@
 import org.junit.Test;
 
 /**
- * Test case for {@link IoCheckedScalar}.
+ * Test case for {@link IoChecked}.
  *
  * @since 0.4
  * @checkstyle JavadocMethodCheck (500 lines)
  */
-public final class IoCheckedScalarTest {
+public final class IoCheckedTest {
```

Comment:
Could we use Throws in these tests?

Reply:
@Iprogrammerr we could, but I spent enough time on this already :)